ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +342,39 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_node_utilization(self, master=True, worker=True):
+        """
+
+        Check CPU and Memory usage and print a table of the data.
+
+        Args:
+            master (bool): if True, check check CPU and Memory usage on master nodes
+            worker (bool): if True, check check CPU and Memory usage on worker nodes
+
+        """
+        usage_memory_table = PrettyTable()
+        usage_memory_table.field_names = ["Node Name", "CPU USAGE adm_top", "CPU USAGE oc_describe",
+                                          "Memory USAGE adm_top", "Memory USAGE oc_describe"]
+
+        if master:
```

Comment:
if you go with what I suggested in the comment above, no need for lines 367-376, as you will simply pass the value of `node_type` to `get_node_resource_utilization_from_adm_top(node_type=node_type)`